package/PartSegImage/image.py

```
@@ -774,6 +776,18 @@ def spacing(self) -> Spacing:
         return tuple(self._image_spacing[1:]) if self.is_2d else self._image_spacing
 
+    @property
+    def time_increment(self) -> float:
+        """time spacing in seconds"""
+        return self._time_increment
+
+    @time_increment.setter
+    def time_increment(self, value: float):
+        """set time spacing in seconds"""
+        if value <= 0:
+            raise ValueError("Time spacing need to be positive")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)